packages/design-system/src/components/Combobox/Combobox.tsx

```diff
@@ -290,6 +281,25 @@ const Combobox = React.forwardRef<ComboboxInputElement, ComboboxProps>(
                 <Box id={intersectionId} width="100%" height="1px" />
               </Viewport>
             </ScrollArea>
+            {creatable || createItemAlwaysVisible ? (
+              <CreateItemContainer>
+                <ComboboxPrimitive.CreateItem
+                  onPointerUp={handleCreateItemClick}
+                  onClick={handleCreateItemClick}
+                  createItemAlwaysVisible={createItemAlwaysVisible}
+                  asChild
+                >
+                  <OptionBox>
+                    <Flex gap={2} justifyContent="space-between">
+                      <Flex gap={2}>
+                        <Plus fill="neutral500" />
+                        <Typography>{createMessage(internalTextValue ?? '')}</Typography>
+                      </Flex>
+                    </Flex>
+                  </OptionBox>
+                </ComboboxPrimitive.CreateItem>
+              </CreateItemContainer>
```

Comment:
We could have something a bit more flexible here by giving the choice for the startIcon for example:

```suggestion
              <CreateItemContainer>
                <ComboboxPrimitive.CreateItem
                  onPointerUp={handleCreateItemClick}
                  onClick={handleCreateItemClick}
                  createItemAlwaysVisible={createItemAlwaysVisible}
                  asChild
                >
                  <OptionBox>
                    <Flex gap={2} justifyContent="space-between">
                      <Flex gap={2}>
                        {creatableStartIcon && (
                          <Box tag="span" aria-hidden>
                            {creatableStartIcon}
                           </Box>
                         )}
                        <Typography>{createMessage(internalTextValue ?? '')}</Typography>
                      </Flex>
                    </Flex>
                  </OptionBox>
                </ComboboxPrimitive.CreateItem>
              </CreateItemContainer>
```